WordPress/src/main/java/org/wordpress/android/ui/jetpack/backup/download/details/BackupDownloadDetailsFragment.kt

```diff
@@ -78,12 +78,12 @@ class BackupDownloadDetailsFragment : Fragment() {
     // todo: annmarie -(REMOVE) this dummy method references layout files that lint says aren't
     // used, but they will be in the next PR because there were too many changes for 1 PR
     private fun dummyAccess() {
-        val buttonLayoutId = R.layout.backup_download_list_button_item
+        val buttonLayoutId = R.layout.jetpack_list_button_item
         val checkboxLayoutId = R.layout.backup_download_list_checkbox_item
-        val descriptionLayoutId = R.layout.backup_download_list_description_item
-        val headerLayoutId = R.layout.backup_download_list_header_item
+        val descriptionLayoutId = R.layout.jetpack_list_description_item
+        val headerLayoutId = R.layout.jetpack_list_header_item
         val subheaderLayoutId = R.layout.backup_download_list_subheader_item
-        val imageLayoutId = R.layout.backup_download_list_image_item
+        val imageLayoutId = R.layout.jetpack_list_icon_item
```

Comment:
Minor (🔍): Maybe you want to update the `image` prefix on the variable with `icon` as well, not sure?

Reply:
Nah, not worried about this. I had another PR ready to go this morning and then Ashita & I switched up the states. So this will be gone in a few hours. Thanks.